DataFormats/L1TrackTrigger/interface/TTTrack.h

```diff
@@ -296,96 +280,119 @@ void TTTrack<T>::setFitParNo(unsigned int nPar) {
 
 template <typename T>
 GlobalVector TTTrack<T>::momentum() const {
-  return theMomentum;
+  return theMomentum_;
 }
 
 template <typename T>
 double TTTrack<T>::rInv() const {
-  return theRInv;
+  return theRInv_;
 }
 
 template <typename T>
 double TTTrack<T>::tanL() const {
-  return theTanL;
+  return theTanL_;
 }
 
 template <typename T>
 double TTTrack<T>::eta() const {
-  return theMomentum.eta();
+  return theMomentum_.eta();
 }
 
 template <typename T>
 double TTTrack<T>::phi() const {
-  return thePhi;
+  return thePhi_;
 }
 
 template <typename T>
 double TTTrack<T>::d0() const {
-  return theD0;
+  return theD0_;
 }
 
 template <typename T>
 double TTTrack<T>::z0() const {
-  return theZ0;
+  return theZ0_;
 }
 
 template <typename T>
 GlobalPoint TTTrack<T>::POCA() const {
-  return thePOCA;
+  return thePOCA_;
 }
 
 /// Chi2
 template <typename T>
 double TTTrack<T>::chi2() const {
-  return theChi2;
+  return theChi2_;
+}
+
+/// Chi2Z
+template <typename T>
+double TTTrack<T>::chi2Z() const {
+  return theChi2_Z_;
 }
 
-/// Chi2z
+/// Chi2XY
 template <typename T>
-double TTTrack<T>::chi2z() const {
-  return theChi2Z;
+double TTTrack<T>::chi2XY() const {
+  return theChi2_XY_;
 }
 
 /// Chi2 reduced
 template <typename T>
 double TTTrack<T>::chi2Red() const {
-  return theChi2 / (2 * theStubRefs.size() - numFitPars);
+  return theChi2_ / (2 * theStubRefs.size() - theNumFitPars_);
 }
 
 /// MVA quality variables
 template <typename T>
 double TTTrack<T>::trkMVA1() const {
-  return theTrkMVA1;
+  return theTrkMVA1_;
 }
 
 template <typename T>
 double TTTrack<T>::trkMVA2() const {
-  return theTrkMVA2;
+  return theTrkMVA2_;
 }
 
 template <typename T>
 double TTTrack<T>::trkMVA3() const {
-  return theTrkMVA3;
+  return theTrkMVA3_;
 }
 
 /// StubPtConsistency
 template <typename T>
 void TTTrack<T>::setStubPtConsistency(double aStubPtConsistency) {
-  theStubPtConsistency = aStubPtConsistency;
+  theStubPtConsistency_ = aStubPtConsistency;
   return;
 }
 
 /// StubPtConsistency
 template <typename T>
 double TTTrack<T>::stubPtConsistency() const {
-  return theStubPtConsistency;
+  return theStubPtConsistency_;
 }
 
+/// StubPtConsistency
```

Comment:
Would be good to correct comment to avoid confusion...

Reply:
done